Dockerfile

```diff
@@ -28,12 +28,18 @@ RUN microdnf update \
     && microdnf remove libcurl-devel gcc python3-devel openssl-devel annobin -y \
     && microdnf clean all
 
+# Need jq for parsing the clowder configuration json in shell scripts
+RUN microdnf install jq -y \
```

Comment:
Any reason not to include this with the other packages above?

Reply:
The dnf installs seemed to be categorized, second one postgres, third one python.  I could add it to the first group though, seems general enough.